### ISSUE TITLE
PHP Parse error

### ISSUE THREAD

**Issue by silas229** [laravel/framework#26177]
- Laravel Version: 5.7.9
- PHP Version: 7.0.3
- Database Driver & Version: 10.1.23-MariaDB-9+deb9u1 

### Description:
I got this error:
`PHP Parse error:  syntax error, unexpected '=' in /var/www/heimdall/vendor/laravel/framework/src/Illuminate/Support/Arr.php on line 388`

### Steps To Reproduce:
- Installing apache2, php, mysql-server, php-mysql
- Cloning [Heimdall](https://github.com/linuxserver/Heimdall) repository
- Set Apache DocumentRoot to Heimdall public/ folder

**Comment by staudenmeir** [comment]
Laravel 5.7 requires PHP 7.1+.

**Comment by silas229** [comment]
oh okay, but how can I get it via Raspbian apt-get?

**Comment by browner12** [comment]
that's a question for another place. I'd look on StackOverflow, or post a question there.

**Comment by silas229** [comment]
Okay, I found an answer. Thanks!

**Comment by rd67** [comment]
@silas229 Can you please post your solution here?

**Comment by scarlet2131** [comment]
could you please tell me how did you fixed that problem here?

**Comment by SyedUmerHasan** [comment]
This problem occurs due to the reason of running different PHP version on the same PC, 
Pre-Requiste,
Laravel 5.7 server requirement is PHP version > 7.1
check this link = https://laravel.com/docs/5.7/installation#server-requirements
For Eg, 
1)I had WAMPP installed with PHP 7.1 and 5.6, with active PHP Version of 7.1, installed globally with environment variable of the user/system
2) I had a laragon running PHP version 7.2, 

I installed laravel using command line and tries to execute **php artisan serve** or any **php artisan command**
command line shows
> PHP Parse error: syntax error, unexpected '=' in /www/computernetworks/vendor/laravel/framework/src/Illuminate/Support/Arr.php on line 388
so basically everything seems to be good as you see your PHP version on the web will be 7.2 but in my case when i check PHP version in command line using 
**php -v** 
it was displaying PHP Version as 7.1

So do one thing do remove the enviroment variable with global PHP environment.
Everything will work fine then.
Note : try to use one PHP version in your PC

**Comment by scarlet2131** [comment]
thank you so much!!
Monisha Ranjan
2nd year, CSE
Indian Institute of Information Technology, Guwahati
